generator.js

```diff
@@ -29,9 +31,11 @@ module.exports = (api) => {
   );
 
   api.extendPackage({
+    scripts: {
+      "serve:standalone": "vue-cli-service serve --mode standalone",
+    },
     dependencies: {
-      "single-spa-vue": "^1.9.0",
-      "systemjs-webpack-interop": "^2.1.2",
```

Comment:
systemjs-webpack-interop is now a dependency of vue-cli-plugin-single-spa instead of the single-spa application.